lib/matplotlib/widgets.py

```diff
@@ -3051,48 +3052,53 @@ def _release(self, event):
         return False
 
     def _onmove(self, event):
-        """Motion notify event handler."""
+        """
+        Motion notify event handler.
+
+        This can do one of four things:
+        - Translate
+        - Rotate
+        - Re-size
+        - Contine the creation of a new shape
+        """
+        xy = np.array([event.xdata, event.ydata])
```

Comment:
```suggestion
        xy = np.array([event.xdata, event.ydata])
```
Side note: I'm wondering if a small internal class `_Point` would make sense for such cases, basically namedtuple-like with arithmetics. So that we can do.

~~~
p = _Point(x, y)
2*p
p + (1, 2)
p.x
~~~
Arithmetics should still be faster than the numpy overhead.